komodoenv/__main__.py

```diff
@@ -144,8 +144,8 @@ def parse_args(args):
     ap.add_argument(
         "--root",
         type=str,
-        default="/prog/res/komodo",
-        help="Absolute path to komodo root (default: /prog/res/komodo)",
+        default="/prog/komodo" if Path("/prog/komodo").is_dir() else "/prog/res/komodo",
+        help="Absolute path to komodo root (default: /prog/komodo on Azure, else /prog/res/komodo)",
```

Comment:
As we are using this tool mainly on onprem, might it make more sense to reword this accordingly?

```
Absolute path to komodo root (default: /prog/res/komodo for onprem, /prog/komodo for Azure)
```